### PR TITLE
Grid msg

### PR DESCRIPTION
**Pull request by joakim-hove**
Small updates in messages printed while creating grid

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: src/opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp [hunk 1]
```diff
@@ -209,17 +211,8 @@ EclipseGrid::EclipseGrid(const Deck& deck, const int * actnum)
       m_multzMode(PinchMode::ModeEnum::TOP),
       m_pinchGapMode(PinchMode::ModeEnum::GAP)
 {
-    OpmLog::info("Creating grid");
     if (deck.hasKeyword("GDFILE")){
 
-        if (deck.hasKeyword("COORD")){
-            throw std::invalid_argument("COORD can't be used together with GDFILE");
```
Comment (atgeirr) [inline src/opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp:216]: Is this not true? Why the change?

Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp:216]: See the very good commit message ;-)

Comment (atgeirr) [reply src/opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp:216]: I see. Good to merge when green.

**Comment by joakim-hove** [comment]
jenkins build this please

File: src/opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp (hunk 1 above)
Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Grid/EclipseGrid.cpp:216]: Seriously - the new functionality provided here: #1996 ensures that the situation can not arise at all.
